packages/ui/src/components/internal/Root.svelte

```diff
@@ -16,8 +16,18 @@
   import LangSelector from './LangSelector.svelte'
   import ThemeSelector from './ThemeSelector.svelte'
   import SearchSelector from './SearchSelector.svelte'
+  import posthog from 'posthog-js'
 
   let application: AnyComponent | undefined
+  if (!window.location.host.includes('127.0.0.1') && !window.location.host.includes('localhost')) {
```

Comment:
Let's assume if not defined it is fine, please remove check. May be we need to test it locally on personal account. Also it is not an error if not defined.

Reply:
Also root is a bad place for it. Please move to dev/prod/configure of platform. So we could decide to use it or not.